stdlib/sqlite3/dbapi2.pyi

```diff
@@ -410,8 +410,6 @@ class InternalError(DatabaseError): ...
 class NotSupportedError(DatabaseError): ...
 class OperationalError(DatabaseError): ...
 
-OptimizedUnicode = str
```

Comment:
This one doesn't seem to exist at runtime or be used in the stub

Reply:
It exists at runtime in 3.6-3.9: https://github.com/python/cpython/blob/3.6/Modules/_sqlite/module.c#L421, https://github.com/python/cpython/blob/636832fb2a4364ca2014aadecf4c35833eff4495/Modules/_sqlite/module.c#L439

Could you add it back with a version guard?

Reply:
Yup, I'd just figured it out from the mypy failures :)

Reply:
I'm not 100% sure whether it's better to do
```python
OptimizedUnicode = str
```
or
```python
OptimizedUnicode: TypeAlias = str
```
or
```python
from builtins import str as OptimizedUnicode
```

Thoughts?